consumers/src/unit-test/java/uk/gov/moj/sdt/cmc/consumers/CMCConsumerGatewayTest.java

```diff
@@ -51,6 +53,15 @@ class CMCConsumerGatewayTest {
     @Mock
     private IIndividualRequest individualRequest;
 
+    @Mock
+    private BreathingSpaceRequest breathingSpaceRequest;
```

Comment:
why the request objects are defined here?

Reply:
Using annotation declaration. simplicity